README.md

```diff
@@ -39,7 +39,12 @@ responses).
 
 ### Sending traces on outgoing requests with Faraday
 
-First, for the Faraday middleware to have the correct trace ID, you should be using the rack middleware in your application.
+First, Faraday has to be part of your Gemfile:
```

Comment:
somehow I merge back with master and got this. Not really a change, that text is in master already